src/h5web/App.tsx

```diff
@@ -45,11 +45,7 @@ function App(): ReactElement {
             selectedNode={selectedNode}
           />
           {isInspecting ? (
-            <MetadataViewer
-              key={JSON.stringify(selectedLink)}
-              link={selectedLink}
-              entity={selectedEntity}
-            />
+            <MetadataViewer link={selectedLink} entity={selectedEntity} />
```

Comment:
Not an ESLint error. I just noticed that we no longer need a key on MetadataViewer since selectedEntity is no longer retrieved asynchronously; it is always in sync with selectedLink.